code/modules/mob/mob_defines.dm

```diff
@@ -227,3 +227,5 @@
 
 	/// A mock client, provided by tests and friends
 	var/datum/client_interface/mock_client
+
+	var/interaction_range = 1 //how far a mob has to be to interact with something, defaulted to 1 tile
```

Comment:
so that the interaction range checks will always function without having to check if something is instead a carbon or a silicon, this will also allow future, or even current mobs, ranges be tweaked more easily. say for example you make a carbon species that has long arms so you want it to be able to use machines 2 tiles away, this will allow you to do that, 

but mostly its just to streamline the checks so i dont have to check "isAI()" and "isrobot()" and "iscarbon()" before doing range checks, it just checks range as is.